build-system/compile/post-closure-babel.js

```diff
@@ -83,24 +82,17 @@ function terserMinify(code) {
  * Apply Babel Transforms on output from Closure Compuler, then cleanup added space with Terser.
  *
  * @param {string} directory directory this file lives in
- * @param {boolean} isEsmBuild
  * @return {!Promise}
  */
-exports.postClosureBabel = function (directory, isEsmBuild) {
-  const babelPlugins = conf.plugins({isPostCompile: true, isEsmBuild});
-
+exports.postClosureBabel = function (directory) {
   return through.obj(function (file, enc, next) {
-    if (path.extname(file.path) === '.map' || babelPlugins.length === 0) {
+    if (path.extname(file.path) === '.map') {
       return next(null, file);
     }
 
     const map = loadSourceMap(file.path);
-    const {code, map: babelMap} = babel.transformSync(file.contents, {
-      plugins: babelPlugins,
-      retainLines: false,
-      sourceMaps: true,
-      inputSourceMap: false,
-    });
+    const caller = {caller: {name: 'post-closure'}};
```

Comment:
I created constant objects to allow for re-use in case there were multiple call-sites in a module that used the exact same config, but this wasn't needed anywhere.

Reverted all to inline objects.